plugins/modules/mssql_script.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2014, Vedit Firat Arig <firatarig@gmail.com>
+# Outline and parts are reused from Mark Theunissen's mysql_db module
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: mssql_script
+short_description: Run simple, file based, or templated mssql scripts
+description:
+   - Run simple, file based, or templated mssql scripts
+options:
+  db:
+    description:
+      - name of the database to select
+    required: true
+    type: str
+  login_user:
+    description:
+      - The username used to authenticate with
+    type: str
+  login_password:
+    description:
+      - The password used to authenticate with
+    type: str
+  login_host:
+    description:
+      - Host running the database
+    type: str
+    required: true
+  login_port:
+    description:
+      - Port of the MSSQL server. Requires login_host be defined as other than localhost if login_port is used
+    default: '1433'
+    type: str
```

Comment:
```suggestion
    type: int
```
Shouldn't it be int?